src/components/BookList.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Booklist = () => {
+  const books = [
+    {
+      id: 1, title: 'Book-1', author: 'Author-1', category: 'Category-1',
+    },
+    {
+      id: 2, title: 'Book-2', author: 'Author-2', category: 'Category-2',
+    },
+    {
+      id: 3, title: 'Book-3', author: 'Author-3', category: 'Category-3',
+    },
+  ];
+
+  return (
+    <div>
+      {books.map((book) => (
+        <div className="book-list" key={book.id}>
+          <h3>{book.title}</h3>
+          <p>{book.author}</p>
+          <p>{book.category}</p>
+          <button type="button" id={book.id}>delete</button>
+        </div>
```

Comment:
As mentioned in the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md#:~:text=Contains%20individual%20book%20state%2C%20as%20well%20as%20a%20button%20to%20delete%20the%20book), a book should be represented in its own component. You should create a new component for an individual book, then you can import it here, and render the list of books with that component. It should look something like this: 
 ```
import Book from './Book';

const BookList = () => {
/* ...  some other code here */

return (
    <div>
        {books.map((book) => (
            <Book item={book} /> 
        ))}
    </div>
);
};
```

Reply:
Thank you for the review! I'm glad you suggested the approach of representing each book as its own component (Book). Using separate components for individual items often leads to more maintainable and reusable code. If you have any further feedback or suggestions, I'd be happy to hear them!